playbooks/roles/certbot/tasks/main.yml

```diff
@@ -54,6 +61,33 @@
     dest: /etc/letsencrypt/renewal-hooks/deploy/restart-nginx.sh
     mode: 0755
 
+# The systemtd certbot renew timer that's included in the certbot snap package does not send an email on failure.
+# For this reason we disable the timer and use a custom cronfile with a MAILTO instead.
+- name: Disable systemd timer
+  systemd:
+    name: snap.certbot.renew.timer
+    state: stopped
+    enabled: no
+
+# Custom cron job for renewing certificates. The only difference from the official snap systemd timer is
+# that this cron job will send an email to ops if any of the certificates fail to renew.
+- name: Create cron job to periodically renew certificates
+  cron:
+    name: certbot renew
+    user: root
+    job: "perl -e 'sleep int(rand(43200))' && /snap/bin/certbot -q renew"
+    minute: '0'
+    hour: '*/12'
+    cron_file: certbot-renew
+
+- name: Set MAILTO variable for certbot renew cron job
```

Comment:
@mtyaka, since this will be useful only in production and sending these emails from other environments to the Ops list adds to the email clutter, can we make this conditional?

Reply:
Yup, good idea. I'll do that.

Reply:
@mtyaka, did you get a chance to address this?

Reply:
Not yet @lgp171188, I'll take care of it later this week.

Reply:
Resolved [here](https://github.com/open-craft/ansible-playbooks/pull/181/files#diff-a664225e5a176729b75a4704ef78d6a6R89)